CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# dbt_salesforce v0.7.1
```

Comment:
Thanks for adding this changelog! Can you add a contributor section so you can be recognized for your contribution? [Leverage this as an example.](https://github.com/fivetran/dbt_hubspot_source/blob/main/CHANGELOG.md#contributors-1)